app/models/manageiq/providers/redhat/infra_manager/provision/cloning.rb

```diff
@@ -72,7 +74,7 @@ def clone_type_by_linked_clone
   end
 
   def clone_type_by_disk_format
```

Comment:
Should this method name change now that we are looking for sparsity not format?

Reply:
oh tnx, missed that. done.